go-controller/pkg/libovsdbops/model_client.go

```diff
@@ -152,11 +152,11 @@ type operationModel struct {
 	DoAfter func()
 }
 
-func onModelUpdatesNone() []interface{} {
```

Comment:
Is there a reason this function cannot be left private? I seem to think it does not need to start with 'O', still.

Reply:
no, just originally I was allowing things outside of libovsdb to pass this, but found later there was really no need. Will make it private again.